driver/src/main/java/org/neo4j/driver/v1/Path.java

```diff
@@ -61,7 +61,7 @@
     Node end();
 
     /** @return the number of segments in this path, which will be the same as the number of relationships */
-    long length();
+    int length();
```

Comment:
2B is a lot for sure - but we've hit this again and again in other places, why constrain the API to that?